scalableminds/webknossos
# adds y-axis scale to the histogram view

### URL of deployed dev instance (used for testing):
- [dev-instance](https://addyscaletohistogram.webknossos.xyz/auth/login)

### Steps to test:
- open a tracing
- open the datasets tab in the settings
- the histogram should now have a scale at the y-axis indicating the logarithmic scale

### Issues:
- fixes #4297

------
